lib/smart_answer_flows/uk-benefits-abroad/questions/do_either_of_the_following_apply.govspeak.erb

```diff
@@ -3,8 +3,12 @@
 <% end %>
 
 <% options(
-  "yes": "Yes",
-  "no": "No"
+  "bereavement_benefits": "Bereavement benefits",
```

Comment:
Similarly to [this comment](https://github.com/alphagov/smart-answers/pull/3180#discussion_r132138290), it would be nice if this list existed in one place.